File: WEBAPI/Models/DTOs/Users/EditUserDTO.cs
namespace Models.DTOs.Internal.Users
{
    public class EditUserDTO
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public decimal? Wage { get; set; }
        public string Phone { get; set; }
    }
}

File: PROJECT/FRONTEND/Common/DTOs/EditUserDTO.ts


export class EditUserDTO { 
    constructor(obj: Partial<EditUserDTO>) {
        Object.assign(this,obj);
    }

    
    public id: number;
    public firstName: string;
    public lastName: string;
    public email: string;
    public wage: number | null;
    public phone: string;
    public roleIds: number[];
}
